index.js

```diff
@@ -424,6 +424,7 @@ class LedgerBridgeKeyring extends EventEmitter {
   _setupIframe () {
     this.iframe = document.createElement('iframe')
     this.iframe.src = this.bridgeUrl
+    this.iframe.allow = `hid 'src'`
```

Comment:
Interesting that this wasn't documented on MDN on the list of features this allowlist applies to, but I did find it in the WebHID spec: https://wicg.github.io/webhid/#permissions-policy

Reply:
oh, yeah, that's how I found that as well. I should have put that in the PR description.